src/components/Identityprovider/identityprovider.component.js

```diff
@@ -105,6 +105,13 @@ export async function authenticate(req, res, next) { // eslint-disable-line comp
       'login_'
     );
 
+    if (state.serviceClient.hideIdentityProviders) {
```

Comment:
Hele denne funktion, **authenticate**, fylder 140 liner, det er godt nok mange.

Har du tid/mulighed for at bryde den ned i mindre funktioner på et tidspunkt?
Evt. starte med at skrive en kommentar til de enkelte if'er man støder på ned igennem funktionen?

Reply:
Har splittet (meget) lidt op og sat lidt kommentarer - men det er stadig 120 linier. Meget at det er parametre til login-siderne, og det er meget rarat at det faktisk er udspecificeret.